### DISCUSSION TITLE
Translatable strings

### DISCUSSION THREAD

**Discussion by LeonarddeR** [statiqdev/Discussions#85] [Ideas]
I wonder whether it would be possible to make CleanBlog translatable. May be this can even be extended to have translatable content in a static website.

**Comment by daveaglick** [comment]
I'm curious what translatable would look like in the context of a specific theme? Do you mean different versions of the CleanBlog theme for different locales? Or some sort of integrated localization support built-in to Statiq?

If the latter, there's some discussion about that possibility here: https://github.com/statiqdev/Statiq.Web/issues/796 - that work has been delayed while I try to finish Statiq Docs, but eventually that issue is what integrated localization support might look like.

**Reply by LeonarddeR** [reply]
I think I'd prefer the latter. Do you think we can continue the discussion in https://github.com/statiqdev/Statiq.Web/issues/796?
